_posts/2016-10-27-journal-of-a-junior-multisite.md

```diff
@@ -0,0 +1,133 @@
+
+
+You will want a 'Homepage' document type, as well as a 'Website Settings'. These are a must - the homepage will be the start node, and in the website settings, we will set each websites unique stylesheet and settings. This isn't necessary but is good to know if each site is going to be different style wise.
+
+
+In my Homepage I have the Umbraco Grid:
+
+![Homepage Settings][homepage]
+
+[homepage]: /assets/images/blog/ums-homepage-settings.png "Homepage Settings"
+
+In my Website Settings, I have some text fields to enter the name or your stylesheet and also fonts.
+
+![Website Settings][website]
+
+[website]: /assets/images/blog/ums-website-settings.png "Website Settings"
+
+###Step 2: Allowing each site to have a unique domain
+
+
+Once you've got Umbraco installed, you will want to go into umbracoSettings.config, and find '<requestHandler>' and set <useDomainPrefixes> to true. This allows you to have unique domains for multiple sites.
+
+![Request Handler][request]
+
+[request]: /assets/images/blog/ums-request-handler.png "Request Handler"
+
+###Step 3: Creating content for each site
+
+
+Now these have been made, we will create them in the Content tab. Remember to set up permissions to allow the Website Settings document to be a child of the Homepage, along with any other Doctypes you have made. It should look a little like the image below:
+
+![Content Tree][content]
+
+[content]: /assets/images/blog/ums-content-tree.png "Content Tree"
+
+
+Now give each site its own unique content! If each site is having its own stylesheet or fonts, then this is where you will add them into the settings. First, you will want to make your stylesheets have been created. In the settings, you can add the file name and your link to the font you have chosen. I'm using google fonts, so I can just add the google link but this can easily be customised to suit you.
+
+
+We used a helper class to get the website settings node:
+<xmp>
+public static IPublishedContent GetWebsiteSettings()
+{
+    var umbracoHelper = new Umbraco.Web.UmbracoHelper(Umbraco.Web.UmbracoContext.Current);
+     Node currentNode = Node.GetCurrent();
+     Node currentHomeNode = new Node(int.Parse(currentNode.Path.Split(',')[1]));
+      var currentHome = umbracoHelper.TypedContent(currentHomeNode.Id);
+      return currentHome.Children.Where(x => x.IsDocumentType("settings")).First();
+}
+</xmp>
+
+And Razor to assign the stylesheet to the template:
+<xmp>
+@{
+    var websiteSettings = SettingsHelpers.GetWebsiteSettings();
+    var styleSheet = websiteSettings.GetProperty("styleSheet").Value;
+    string fonts = websiteSettings.GetProperty("fontReference").Value.ToString();
```

Comment:
btw I think more elegant style would be websiteSettings.GetPropertyValue<string>("fontReference");